kolena/workflow/test_run.py

```diff
@@ -487,6 +486,7 @@ def _start_server_side_evaluation(self) -> None:
 
 
 @validate_arguments(config=ValidatorConfig)
+@with_event(event_name=EventAPI.Event.EXECUTE_TEST_RUN)
```

Comment:
Double checking, it seems like we already decorated `TestRun.run`, then we should not add here.

Reply:
Discussed offline, made [a commit](https://github.com/kolenaIO/kolena/pull/299/commits/178a639d383304c7d8ea7c459c6b3235525c55e0) to leave at the `TestRun.run` level so the event will not missed if a user is not using this `test()` interface.